README.md

```diff
@@ -1862,7 +1922,23 @@ attributes_to_string(attrs)
 ## Template tag syntax
 
 All template tags in django_component, like `{% component %}` or `{% slot %}`, and so on,
-support extra syntax that makes it possible to write components like in Vue or React.
+support extra syntax that makes it possible to write components like in Vue or React (JSX).
+
+### Self-closing tags
```

Comment:
I realized that it would be good to document the self-closing tags in this section too.